api/db/populate-messages.js

```diff
@@ -13,6 +17,21 @@ const createMessage = async (messageParams) => {
   });
   await message.save();
 
+  const user =
+    message.friendship.receivant.toString() === message.user.toString()
+      ? message.friendship.receivant
+      : message.friendship.requestant;
```

Comment:
This should be the other way around.

Also, maybe it would help make it a bit more understandable to write `message.user.toString() === message.friendship.receivant.toString()` instead of as it's written now, but feel free to keep it as is!